src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -59,10 +59,6 @@ public function getMarking($subject)
     {
         $marking = $this->markingStore->getMarking($subject);
 
-        if (!$marking instanceof Marking) {
```

Comment:
OK. Im still not convinced though. Anyway hope to get some more feedback/thumbs this day then, and i'll have a look tonight. However i feel my arguments are not taken seriously.

Dont get me wrong;
> on the PR as a whole

From this debate on there's no clear yes/no till now, if you ask me, besides two :+1: for the the PR. Anyway, to move on, ill revert it from a "out of scope of this PR" point of view.